Scrum-63

Ale021217 opened the pull request:
Revisión y correción del swagger de post y put de zones

Ale021217 committed:
Scrum-63

Revisión y correción del swagger de post y put de zones

Ale021217 committed:
Scrum-63

Correción de la implementación de "Location" por "description"

ebarahonautn commented on swagger.yml:
@Ale021217 

Revisar y arreglar el error en swagger.yml

![Screenshot 2024-06-24 at 10 05 36 AM](https://github.com/utn-integrador-III/student-service-api/assets/80927814/a89d4155-aff2-4efb-9947-0b31d6f43d77)

Ale021217 committed:
Scrum-63

Error actualizado, el error lo comprobé en swagger editor y ya vi cual era el error y ya corre bien el swagger.yml.
